examples/old_asyncio_example.py

```diff
@@ -1,14 +1,14 @@
 import asyncio
```

Comment:
Why'd you update this file? This is now identical to `simple_example.py`? This file isn't really relevant anymore, but it's a relic of when I was keen on keeping python 3.4 support for this library (which is why I avoid the async/await keywords throughout the codebase). There's probably no reason to do this anymore so we could update all the @asyncio.couroutine and yield from's to async and awaits if we wanted - but probably not worth the effort.

Reply:
Oh I didn't realize that that was for the old asyncio support, I did a simple find and and replace operation. I would have liked to also keep the python 3.4 support but with python 3.10 [removes](https://docs.python.org/3/library/asyncio-task.html#asyncio.coroutine) this version so it might not be worth supporting it. Your call.

Reply:
Hmm should probably update to support python 3.10, I think supporting modern versions is more of a priority than supporting legacy versions.

But let's not do that under this PR, I'll make a new issue for it

Reply:
okay I'll revert it